src/Apps/Order/Routes/Shipping2/FulfillmentDetails.tsx

```diff
@@ -164,16 +185,16 @@ export const FulfillmentDetails: FC<FulfillmentDetailsFormProps> = ({
       onSubmit={handleSubmit}
     >
       <FulfillmentDetailsFormLayout
-        onSubmit={props.onSubmit}
         renderMissingShippingQuotesError={
           props.renderMissingShippingQuotesError
         }
         order={props.order}
+        onSubmit={props.onSubmit}
         me={props.me}
         setFulfillmentFormHelpers={props.setFulfillmentFormHelpers}
         active={active}
         verifyAddressNow={verifyAddressNow}
-        addressVerificationComplete={() => setVerifyAddressNow(false)}
+        addressVerificationComplete={handleVerificationComplete}
```

Comment:
`addressVerificationComplete` => `onAddressVerificationComplete`

Reply:
I was following [what I understood the convention to be](https://stackoverflow.com/a/60048346/4409186), which is:
- expose props from your component the same way html does, with `on*`
- When making your own functions to pass as props, use `handle*`

So you might have a component that accepts an `onSubmit` prop, and passes that into a button: `<Button onClick={props.onSubmit}>Submit</Button>`, or you might have to add a little more to it:
```ts
const handleSubmit = () => {
}

Reply:
Your explanation up above is totally correct and how it is done. 

However, the implementation in this PR is not following the guidance here and lacks appropriate naming metadata. 

For example, imagine you've never seen this code before. 

```
addressVerificationComplete
```
What comes  to mind here? Is it a `boolean` value? is it a function? Callback code that is passed **downward** in the react should always be prefixed with `on`, so that -- at a glance -- all ambiguity is reduced. 

Following the above exercise, lets start over. Imagine you're seeing this code for the first time: 

```
onAddressVerificationComplete
```
Is there any question whatsoever what this thing is? Its obviously a function callback. 

And so:

```
onAddressVerificationComplete={handleAddressVerificationComplete}
```

Local functions are `handleFoo`, which are passed props prefixed with `onFoo`; internal to the the child component, it then becomes clear what's going on without needed to read around all the code.

Reply:
oh yeah, you are right. the prop is named wrong.